website/docs/prysm-usage/fee-recipient.md

```diff
@@ -0,0 +1,76 @@
+---
+id: fee-recipient
+title: Fee Recipient
+sidebar_label: fee recipient
+---
+
+# Fee Recipient
+
+Fee Recipient is the feature that allows users receive gas fees when proposing blocks. These gas rewards are some times know as the inclusion rewards.
+
+:::tip Post Bellatrix Fork ( The Merge )
+This feature will only activate Post Bellatrix Fork. Currently stakers do not receice gas fees from proposing blocks.
+:::
+
+:::warning Known Release Bug
+Setting a default fee recipient through the validator client is currently not working in the upcoming branch a fix was created and merged into develop for the following release. please refer to [#10555](https://github.com/prysmaticlabs/prysm/pull/10555)
+:::
+
+:::warning Be aware of your Fee Recipient Settings!!
+
+Protect and review your Fee Recipient settings.
+
+Fee Recipient settings will default to the burn address ( 0x000....0 ) so it is important to set it to a hex address where you can be paid this. 
```

Comment:
```suggestion
If you don't configure your Fee Recipient settings, the Fee Recipient address will default to the burn address (`0x000....0`). Be sure to specify a valid Ethereum wallet address (that you own) as the fee recipient. 
```
**assumption**: most users will recognize "Ethereum wallet address"; "hex address" is ambiguous.
meta-feedback: we want to use precise and unambiguous terminology when possible.

Reply:
Recommend moving this down to the step at which the address is specified within the config.